web/themes/custom/recipe_theme/src/sass/content/_node.landing-page.scss

```diff
@@ -3,3 +3,31 @@
   margin-bottom: 3rem;
   opacity: 0.5;
 }
+
+.landing-page__body {
+  text-align: center;
+  @include for-phone-only {
```

Comment:
If these are the default rules, do you need to specify that these are explicitly for very small screens?

Reply:
Sorry, I'm not sure what the issue is here. Could you please elaborate?